.docker/bin/dsh

```diff
@@ -78,13 +102,18 @@ get_project_root_path ()
 # dummy function to check that command is run along settings.php
 get_drush_path ()
```

Comment:
This function change is unrelated to update. It's for "drush functions will work even outside of docroot"